docs/reference/style-guide.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Documentation style guide
+authors: Tim McMackin
+last_update:
+  date: 9 January 2024
+---
+
+## Overall Tezos-related issues
+
+- Tezos is decentralized.
+There is no official Tezos documentation, no official Tezos strategy, and no official entity in charge of Tezos.
+
+- Do not compare Tezos to other specific blockchains.
+You can say that Tezos has advantages over other blockchains, but don't say that Tezos is better than or does  things differently from another specific blockchain.
+
+- Use "tez" to describe the currency instead of "XTZ" unless there is a specific reason to use the ISO code/ticker symbol "XTZ," such as in accounting systems, exchange rates with other currencies, and anything that needs a standardized code.
+
+- Avoid using the ꜩ glyph in text.
+
+- Do not use the term "initial coin offering (ICO)" or refer to Tezos "investors."
```

Comment:
US English puts the punctuation inside the quotes unless readers might accidentally include the punctuation erroneously, as in:
> In the XYZ field, enter the value "ABC".
https://style.mla.org/punctuation-and-quotation-marks/
Do you do differently in the Octez docs?

Reply:
I didn't know until today, but we generally follow the Oxford style :-) 

BTW, the example you're giving is not an error IMO, but rather breaks the (academic) rule to avoid ambiguity, see section "If the Rule Doesn’t Fit, Break It" in <https://cmosshoptalk.com/2020/10/20/commas-and-periods-with-quotation-marks/>.

Reply:
Yes, that was what I was trying to express: break the rule if following the rule would lead the reader to make a mistake.

Reply:
Oh, by reading that again, I see your point, sorry! So we agree.